packages/core/src/shared/utilities/cacheUtils.ts

```diff
@@ -7,6 +7,9 @@ import * as vscode from 'vscode'
 import { dirname } from 'path'
 import { ToolkitError, isFileNotFoundError } from '../errors'
 import { SystemUtilities } from '../systemUtilities'
+import { promises as fsPromises } from 'fs'
+import { isWeb } from '../../common/webUtils'
+import crypto from 'crypto'
```

Comment:
Not a blocker, but potential followup: does this import cause any problem for web?

Also: could we use `getNonexistentFilename(..., max=0)` instead?  https://github.com/aws/aws-toolkit-vscode/blob/49828d79371bdf440f616b2ae9ce9aa9e90a05c3/packages/core/src/shared/filesystemUtilities.ts#L200